main/ertp/api/amount-math.md

```diff
@@ -1,26 +1,26 @@
-# Amount Math
+# AmountMath
 
 Logic for manipulating `amounts`.
 
 ## Importing and Using AmountMath
 
-To use the `amountMath` library, import it from ERTP:
-- `import { amountMath } from '@agoric/ertp';`
+To use the `AmountMath` library, import it from ERTP:
+- `import { AmountMath } from '@agoric/ertp';`
 ## AmountMath Kinds
 
-There are two different kinds of `amountMath`, each of which implements all the 
-methods shown on this page. You only have to specify the `amountMath` kind when 
+There are two different kinds of `AmountMath`, each of which implements all the 
+methods shown on this page. You only have to specify the `AmountMath` kind when 
 creating its associated `issuer`.
 
-The two `amountMathKinds` each implement all of the same API methods 
-(i.e. `amountMath` methods are polymorphic). We recommend you import 
+The two `AmountMathKinds` each implement all of the same API methods 
+(i.e. `AmountMath` methods are polymorphic). We recommend you import 
 the `MathKind` values from `@agoric/ERTP` instead of making the strings yourself. 
 
 - `MathKind.NAT` (`nat`): Used with fungible assets. `amount` `values` are natural numbers (non-negative BigInts).
 - `MathKind.SET` (`set`): Used with non-fungible assets. `amount` `values` are objects or records with multiple properties.
 
-Use `makeIssuerKit(allegedName, amountMathKind, displayInfo)` to specify which `amountMathKind` 
-your contract uses. The second parameter, `amountMathKind` is optional and 
+Use `makeIssuerKit(allegedName, AmountMathKind, displayInfo)` to specify which `AmountMathKind` 
```

Comment:
Thanks so much for doing this renaming. I think we're going to have merge conflicts with the AssetKind branch, and also, this parameter will still remain lowercase: `assetKind`. We might have to redo this PR after the AssetKind branch lands, with some more things remaining lowercase.